src/components/MultiHopTrade/hooks/useGetTradeQuotes.tsx

```diff
@@ -103,16 +109,18 @@ export const useGetTradeQuotes = () => {
         error: lifiQuery.error,
         swapperName: SwapperName.LIFI,
       },
-    ].map(result => {
-      const quote = result.data && result.data.isOk() ? result.data.unwrap() : undefined
-      const inputOutputRatio = quote
-        ? getInputOutputRatioFromQuote({
-            quote,
-            swapperName: result.swapperName,
-          })
-        : -Infinity
-      return Object.assign(result, { inputOutputRatio })
-    })
+    ]
+      .filter(result => !isCrossAccountTrade || isCrossAccountTradeSupported(result.swapperName))
```

Comment:
This would be more readable if it were something like:

```suggestion
      .filter(result => !isCrossAccountTrade || swapperSupportsCrossAccountTrade)
```

so reading it in one pass is like "if it's either not a cross account trade, or if it is but the swapper supports cross account trades".